<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenLandSettings</class>
 <widget class="QDialog" name="OpenLandSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>439</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Настройки openLand</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/plugins/openland/icons/settings.png</normaloff>:/plugins/openland/icons/settings.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <widget class="QWidget" name="tabDataBase">
      <attribute name="title">
       <string>Данные</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Базовый проект и параметры БД</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Сервер</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditProjectFile">
            <property name="text">
             <string>Не указан</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Автоматически обновляемый&lt;br /&gt;файл проекта</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBoxIsUseDefaultConnection">
            <property name="text">
             <string>Использовать для обновления проекта openLand.qgs</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Порт</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEditDataBase">
            <property name="text">
             <string>openland</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditPort">
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly</set>
            </property>
            <property name="text">
             <string>5432</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Пользователь</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>База данных</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEditUserName">
            <property name="text">
             <string>openlanduser1</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Пароль</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEditUserPassword">
            <property name="text">
             <string>user1pass</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditServer">
            <property name="text">
             <string>localhost</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButtonFileSelect">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/plugins/openland/icons/find_search.png</normaloff>:/plugins/openland/icons/find_search.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>XML схемы</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Номер версии XML схемы межевого плана</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxVersionXmlMp">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>4</number>
            </property>
            <property name="maximum">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Каталоги, используемые по умолчанию</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Каталог для создаваемых&lt;br /&gt;XML файлов МП&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPathToXmlCreate">
            <property name="text">
             <string>Не указан</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDirectoryOpen">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/plugins/openland/icons/find_search.png</normaloff>:/plugins/openland/icons/find_search.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>lineEditPathToXmlCreate</zorder>
         <zorder>label_14</zorder>
         <zorder>pushButtonDirectoryOpen</zorder>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInterface">
      <property name="maximumSize">
       <size>
        <width>439</width>
        <height>268</height>
       </size>
      </property>
      <attribute name="title">
       <string>Интерфейс</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>При выборе одного полигона слоя  «Участок» производить операции с</string>
         </property>
         <widget class="QRadioButton" name="radioButtonMultiCont">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>188</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Земельным участком полностью</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButtonOneCont">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>48</y>
            <width>411</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Только выбранным контуром МЗУ (входящим в ЕЗП, ЧЗУ)</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxIsSortAttributes">
         <property name="text">
          <string>Сортировать атрибуты земельного участка в списке </string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxIsPointsShow">
         <property name="text">
          <string>Отображать точки и границы ЗУ в списке окна атрибутов</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxIsShowOnlyUserData">
         <property name="text">
          <string>Отображать МП и документы только текущего пользователя БД</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCoordSystem">
      <attribute name="title">
       <string>Система координат</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Система координат</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxMSK"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonMultiCont</tabstop>
  <tabstop>radioButtonOneCont</tabstop>
  <tabstop>checkBoxIsSortAttributes</tabstop>
  <tabstop>checkBoxIsPointsShow</tabstop>
  <tabstop>checkBoxIsShowOnlyUserData</tabstop>
  <tabstop>checkBoxIsUseDefaultConnection</tabstop>
  <tabstop>lineEditProjectFile</tabstop>
  <tabstop>pushButtonFileSelect</tabstop>
  <tabstop>lineEditServer</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>lineEditDataBase</tabstop>
  <tabstop>lineEditUserName</tabstop>
  <tabstop>lineEditUserPassword</tabstop>
  <tabstop>spinBoxVersionXmlMp</tabstop>
  <tabstop>lineEditPathToXmlCreate</tabstop>
  <tabstop>pushButtonDirectoryOpen</tabstop>
  <tabstop>comboBoxMSK</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpenLandSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpenLandSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
